packages/networks/types.ts

```diff
@@ -72,6 +72,8 @@ export type CosmosNetworkInfo = NetworkInfoBase & {
   daoVotingCw4CodeId?: number;
   cwAdminFactoryContractAddress?: string;
   coreDAOAddress?: string;
+  cwAddressListContractAddress?: string;
+  cwAddressListCodeId?: number;
```

Comment:
please add a NetworkFeature if you need this in this PR

Reply:
Done